lession9_js/main.js

```diff
@@ -0,0 +1,176 @@
+/*
+Bài 1
+Viết một chương trình nhập lương nhân viên, tính thuế thu nhập và lương ròng
+ (số tiền lương thực sự mà nhân viên đó nhận được). 
+ Với các thông số giả sử như sau (không theo luật lương, 
+ chỉ là con số giả sử để dễ tính toán):
+•	30% thuế thu nhập nếu lương là 15 triệu.
+•	20% thuế thu nhập nếu lương từ 7 đến 15 triệu.
+•	10% thuế thu nhập nếu lương dưới 7 triệu.
+ */
+const salary = (s) => {
+  let t 
+  if (s > 15*Math.pow(10,6)) {
+    t = 0.3
+  }
+  else if (s >= 7*Math.pow(10,6) && s <= 15*Math.pow(10,6)) {
+    t = 0.2
+  }
+  else if (s > 0 && s < 7*Math.pow(10,6)) {
+    t = 0.1
+  }
+  else {
+    return `luong nhap sai`
+  }
+  return `thue thu nhap(${t*100}%):${t * s} \nluong rong:${s - t * s}`
+}
+/*
+Bài 2
+Viết chương trình nhập tuổi và in ra kết quả nếu tuổi học sinh đó 
+không đủ điều kiện vào học lớp 10.
+Biết tuổi vào lớp 10 của học sinh là 16.
+ */
+const condition = (age) => {
+  if(age>=16)
+  {
+    return `du dieu kien vao lop 10`
+  }
+  else if(age>0 && age<16)
+  {
+    return `khong du dieu kien vao lop 10`
+  }
+  else
+  {
+    return `nhap sai tuoi`
+  }
+}
+
+ /*
+Bài 3
+Viết chương trình nhập một số nguyên bất kỳ và 
+in kết quả ra màn hình để thông báo cho người dùng biết số đó lớn hay nhỏ hơn 100.
+ */
+const checkNumber = (num) => {
+  if(num>100) {
+    return `so lon hon 100`
+  }
+  else if(num ==100) {
+    return `so do la 100`
+  }
+  else {
+    return `so nho hon 100`
+  }
```

Comment:
```suggestion
  if (num > 100) {
    return `so lon hon 100`
  } else if (num === 100) {
    return `so do la 100`
  } else {
    return `so nho hon 100`
  }
```
Đọc kỹ các comment bên trên
Khi so sánh hãy dùng `===` thay cho `==` để chặt chẽ hơn